packages/medulas-react-components/src/components/PageLayout/index.tsx

```diff
@@ -17,24 +17,27 @@ interface Props extends StyleProps {
 
 interface StyleProps {
   readonly color?: 'white' | 'transparent';
+  readonly minHeight?: string;
```

Comment:
Better use `string | number` instead of `string`. Because it possible to use `number` in case of pixel height.

Reply:
But in order to set pixel height you usually do "500px", not "500", and that way it supports every value for height, such as "auto", "100vh", "50rem".

Reply:
But material UI allows you to set it as number too. Just for consistency.

Reply:
It is even possible to set breakpoints with values for the property (you just need to provide specific order as value). This is not required in our case but it is possible. See https://material-ui.com/system/basics/ (https://material-ui.com/system/basics/#demo)